.github/changeset-presets/bump-react.md

```diff
@@ -3,6 +3,7 @@
 '@aws-amplify/ui-react': minor
 '@aws-amplify/ui-react-core': patch
 '@aws-amplify/ui-react-native': major
+'@aws-amplify/ui-react-liveness': minor
```

Comment:
Can we consolidate the changeset into a single one? Also, don't think `ui-react-native` needs a major bump.

Reply:
Hm do you mean combining `bump-react.md` and `bump-versions.md`? If so I'd prefer not to do that in this PR since this is just appending to our existing verion bumps

Reply:
Concerned about `'@aws-amplify/ui-react-native': major`  - don't think that's right

Reply:
Think that is currently what we have in `ui/main` though right?

Reply:
My bad, this is the presets not an actual changeset. Ignore my comments. That being said...do we need a changeset for this PR?

Reply:
we probably do yep!